travispy/entities/hook.py

```diff
@@ -30,3 +31,29 @@ class Hook(Entity):
         'private',
         'admin',
     ]
+
+    def _set_hook(self, flag):
+        '''
+
+        '''
+        url = self._session.uri + '/hooks/{}'.format(self.id)
+        data = {
+            "hook": {
+                "active": flag
+            }
+        }
+
+        response = self._session.put(url, json = data)
+
+        return response.status_code == 200
+
+
+    def disable(self):
```

Comment:
``disable`` and ``enable`` are good names but to publish them into the generated documentation it is necessary to add some explanation.